.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
can we leave `config/database.yml` tracked and use environment variables for `production` environment?

I say so to avoid the `copy to database.yml` thing